docs/reference/integrations.md

```diff
@@ -0,0 +1,27 @@
+# Integrations
+
+### agent
+
+_Interface_: jenkins_agent_v0
+_Supported charms_: [jenkins-agent-k8s](https://charmhub.io/jenkins-agent-k8s),
+[jenkins-agent](https://charmhub.io/jenkins-agent)
+
+Jenkins agents provide a way to perform tasks scheduled by the Jenkins server. Jenkins agents are
+used to distribute workload across multiple containers, allowing parallel execution of jobs.
+
+Example agent relate command: `juju relate jenkins-k8s:agent jenkins-agent-k8s:agent`
```

Comment:
s/relate/integrate "relations" are old terminology
s/juju relate/juju integrate too

Reply:
Thanks, I updated this for all relevant files